detector/data_reader.py

```diff
@@ -0,0 +1,52 @@
+import torchvision
+import os
+import torch
+import pandas as pd
+
+
+def create_label_list(data_path):
+    list_names = []
+    list_target = []
+    for f in os.listdir(data_path + "\\good"):
+        list_names.append("\\good\\" + f)
+        list_target.append(0)
+    for f in os.listdir(data_path + "\\defect"):
+        list_names.append("\\defect\\" + f)
+        list_target.append(1)
+
+    df1 = pd.DataFrame(list_names, columns=["file_name"])
+    df1["label"] = pd.DataFrame(list_target)
+    return df1
+
+
+def DataReader(
```

Comment:
```suggestion
def read_data(
```
Functions name should be snake-case and start with a verb. Camel-case is for classes/objects.

Reply:
Function has been renamed to "read_data"